board/xen/xenguest_arm64/xenguest_arm64.c

```diff
@@ -63,6 +66,90 @@ static int get_next_memory_node(const void *blob, int mem)
 	return mem;
 }
 
+#ifdef CONFIG_VIRTIO_BLK
+#ifdef CONFIG_VIRTIO_PCI
+static void add_pci_mem_map(const void* blob, int* cnt)
+{
+	struct fdt_resource reg_res;
+	int node = -1, len = 0, cells_per_record = 0, max_regions = 0;
+	int pci_addr_cells = 0, addr_cells = 0, size_cells = 0;
+
+	if ((*cnt) >= MAX_MEM_MAP_REGIONS)
+		return;
+	node = fdt_node_offset_by_prop_value(blob, -1,"compatible", "pci-host-ecam-generic", 22);
```

Comment:
So last parameter is basically `sizeof("pci-host-ecam-generic")`?

Reply:
Done